tensorflow/python/BUILD

```diff
@@ -1730,7 +1730,6 @@ tf_cuda_library(
     srcs = ["client/tf_session_helper.cc"],
     hdrs = ["client/tf_session_helper.h"],
     deps = [
-        ":construction_fails_op",
```

Comment:
It breaks building with bazel 0.4.5. I believe the error is due to `:construction_fails_op` being marked as "test only".